README.md

```diff
@@ -328,6 +328,86 @@ This process is inspired by Bitcoin. The difference is that we use a single SHA3
   did:abt:zNKtCNqYWLYWYW3gWRA1vnRykfCBZYHZvzKr
   ```
 
+### Declare DID
+
+Declaring a DID is done by sending a declare transaction to the ABT network. The following is a sample transaction.
+
+```json
+{
+  "code": "OK",
+  "hash": "36BBCA0115A52C0F43C42E84CAE368481A0F32B218380721E3DD2B0456D1D294",
+  "height": 1521,
+  "index": 224,
+  "tags": [
+    {
+      "key": "moniker",
+      "value": "Wuckert"
+    }
+  ],
+  "tx": {
+    "chainId": "forge",
+    "from": "z1RMrcjJVwuohBoqAsPaVvuDajQi1fDo8Qx",
+    "itx": {
+      "__typename": "DeclareTx",
+      "data": null,
+      "moniker": "Wuckert",
+      "pk": "IWNMqz5IdsqxO0x9iqdlSfMvPkchVc3un8mmLXT_GcU",
+      "type": {
+        "address": "BASE58",
+        "hash": "SHA3",
+        "pk": "ED25519",
+        "role": "ROLE_ACCOUNT"
+      }
+    },
+    "nonce": 1,
+    "signature": "E_BkPhw-WUpkTk5nn_WF4z-8huOBqjl-3vQ122TYCDQiahFlklVJT3I7YUwr8d-pi_mqMM0JKWB06ayJh3gODQ",
+    "signatures": []
+  }
+}
+```
```

Comment:
Remove necessary field that is related to our chain.  only the part relavant to DID document